src/Canvas.js

```diff
@@ -168,7 +168,7 @@ var Canvas = React.createClass({
                         || nextProps.columns !== this.props.columns
                         || nextProps.width !== this.props.width
                         || nextProps.cellMetaData !== this.props.cellMetaData
-                        || !shallowEqual(nextProps.style, this.props.style);
+                        || shallowCloneObject(nextProps.style) !== shallowCloneObject(this.props.style);
```

Comment:
`shallowEqual` is still available but moved to a separate add-on package; i figured we could just use `shallowCloneObject` to achieve the same thing here? we already had it and it's one less package...